client/my-sites/site-settings/form-general.jsx

```diff
@@ -529,6 +535,10 @@ const connectComponent = connect(
 		return {
 			siteIsJetpack,
 			siteSlug: getSelectedSiteSlug( state ),
+			supportsLanguageSelection:
+				! siteIsJetpack ||
+				( isJetpackMinimumVersion( state, siteId, '5.8-beta' ) &&
```

Comment:
I had to change this to `5.8-alpha` to work in my env.

Reply:
Thank for clarifying how the versioning will work. Changed it :)